Recrutify/Recrutify.Services/Validators/CandidatesIdValidator.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentValidation;
+using Recrutify.DataAccess.Models;
+using Recrutify.DataAccess.Repositories.Abstract;
+using Recrutify.Services.DTOs;
+
+namespace Recrutify.Services.Validators
+{
+    public class CandidatesIdValidator : AbstractValidator<CreateBullFeedbackTestDTO>
+    {
+        public CandidatesIdValidator(ICandidateRepository candidateRepository)
+        {
+            RuleFor(x => x)
+                .NotNull()
+                .MustAsync(async (model, cancellation) => await CheckedId(candidateRepository, model))
```

Comment:
don't pass repository to rule, just write MustAsync(CheckedId)

Reply:
+